loleaflet/src/layer/tile/CanvasSectionContainer.ts

```diff
@@ -584,7 +584,7 @@ class CanvasSectionContainer {
 		for (var i: number = 0; i < this.sections.length; i++) {
 			if (this.sections[i].zIndex === section.zIndex && this.sections[i].name !== section.name) {
 				var currentTop =  this.sections[i].myTopLeft[1] + this.sections[i].size[1];
-				if (currentTop > maxY && currentTop < section.myTopLeft[1]) {
+				if (currentTop > 0 && currentTop > maxY && currentTop < section.myTopLeft[1]) {
```

Comment:
Now i read your commit messages again, thank you for that catch :) 

Maybe we can add a public variable to sections showing if their size and value are calculated or not. And then we can check for that variables in appropriate locations.

I thought of using very large negative numbers but that would be a workaround. Using a variable like isCalculated or something of your choice might be better.

Reply:
My reasoning is : currentTop is the sum of topleft and size, so this can be <=0 only if at least topleft is <=0  size <=0 which is only possible if locateSection() has not run yet (or the client code purposefully set some negative values which does not make sense). But having a boolean var around is handy as it will help reduce those N^3 or worse loops (indirectly) over sections in other places too I think. Thanks.

Reply:
Thank you for the fix. Just one note: We should set isLocated to false before every onResize event. Then inside locateSections function, we should set their isLocated variable to true again one by one. locateSections function handles section sizes and positions from scratch whenever it is called.

Reply:
I think i need this fix :) Since i have corner header at position (0, 0).

@kendy @dennisfrancis  Can we merge it?